jscomp/others/js_json.ml

```diff
@@ -60,6 +59,29 @@ let test (type a) (x : 'a) (v : a kind) : bool =
   | Array -> Js_array.isArray x 
   | Object -> (Obj.magic x) != Js.null && Js.typeof x = "object" && not (Js_array.isArray x )
 
-
 external parse : string -> t = "JSON.parse" [@@bs.val]
 (* TODO: more docs when parse error happens or stringify non-stringfy value *)
+
+let null : t = Obj.magic Js.null 
+
+let string s : t = Obj.magic s
+
+let number f : t = Obj.magic f
+
+let number_of_int i : t = Obj.magic i
+
+let boolean b : t = Obj.magic (Js_boolean.to_js_boolean b)
```

Comment:
I think we could provide the option to encode boolean as int but I still prefer to leave this API as is. The reason is that the caller might really want to put a `true` value in the JSON and it does not own the other side. 

Encoding bool as int decision is better left to the caller I think.